diff --unified --recursive --text ocl-icd.orig/icd_windows.c ocl-icd.new/icd_windows.c
--- ocl-icd.orig/icd_windows.c	2019-07-18 13:13:51.327729100 -0700
+++ ocl-icd.new/icd_windows.c	2019-07-18 13:15:57.654485800 -0700
@@ -53,13 +53,13 @@
 BOOL CALLBACK khrIcdOsVendorsEnumerate(PINIT_ONCE InitOnce, PVOID Parameter, PVOID *lpContext)
 {
     LONG result;
-    const char* platformsName = "SOFTWARE\\Khronos\\OpenCL\\Vendors";
+    const char* platformsName = "Software\\Khronos\\OpenCL\\Vendors";
     HKEY platformsKey = NULL;
     DWORD dwIndex;
 
     KHR_ICD_TRACE("Opening key HKLM\\%s...\n", platformsName);
     result = RegOpenKeyExA(
-        HKEY_LOCAL_MACHINE,
+        HKEY_CURRENT_USER,
         platformsName,
         0,
         KEY_READ,
